spec/features/pages/show_page_spec.rb

```diff
@@ -88,6 +91,10 @@
     expect(page).to have_content('It is pretty cool too')
   end
 
+  it 'Should display the accordion component' do
+    expect(page).to have_content('FAQ 1')
+  end
+
```

Comment:
Can we add a test that checks to make sure that when the anchor link name is added to the url, the page jumps to the correct top position?

Reply:
I looked in to this...  and found https://stackoverflow.com/questions/18948687/capybara-how-to-test-link-to-different-section-of-same-page .. so not sure if this is possible.